data/tools/wmllint

```diff
@@ -1531,7 +1532,7 @@ def global_sanity_check(filename, lines):
             precomment = nav.text
             if '#' in nav.text:
                 precomment = nav.text[:nav.text.find("#")]
-            if "{SPECIAL_NOTES}" in precomment:
+            if "{SPECIAL_NOTE" in precomment:
```

Comment:
Is this condition supposed to match both the new forms and the deprecated forms? If so, I'd recommend a comment.